advanced/Scripts/webpage.sh

```diff
@@ -456,6 +460,7 @@ main() {
 		"setdns"            ) SetDNSServers;;
 		"setexcludedomains" ) SetExcludeDomains;;
 		"setexcludeclients" ) SetExcludeClients;;
+		"halt"              ) Halt;;
```

Comment:
Thinking of what I wrote above you might want to change the name to `shutdown` (not a hard requirement, though).